### PR TITLE
Features/ajax

### PR DESCRIPTION
**Pull request by Draqe**
@nzdrml

### CODE REVIEW

File: app/controllers/tweets_controller.rb [hunk 1]
```diff
@@ -2,6 +2,11 @@ class TweetsController < ApplicationController
 
   before_action :authorize
 
+  def index
+    tweets = Tweet.all
+    render json: tweets
```
Comment (nzdrml) [inline app/controllers/tweets_controller.rb]: When and where do you call `index` to retrieve this data?

File: app/controllers/tweets_controller.rb [hunk 2]
```diff
@@ -2,6 +2,11 @@ class TweetsController < ApplicationController
...
+  end
+
   def new
```
Comment (nzdrml) [inline app/controllers/tweets_controller.rb]: Do you even use the `new` action

File: app/controllers/tweets_controller.rb (hunk 1 above)
Comment (Draqe) [reply app/controllers/tweets_controller.rb]: Sorry, I did not see this

File: app/controllers/tweets_controller.rb (hunk 2 above)
Comment (Draqe) [reply app/controllers/tweets_controller.rb]: Yes, the new action is used

**Comment by Draqe** [comment]
@nzdrml

**Pull request merged**